test/test_prototype_transforms_functional.py

```diff
@@ -508,6 +508,22 @@ def test_unkown_type(self, info):
         with pytest.raises(TypeError, match=re.escape(str(type(unkown_input)))):
             info.dispatcher(unkown_input, *other_args, **kwargs)
 
+    @make_info_args_kwargs_parametrization(
+        [
+            info
+            for info in DISPATCHER_INFOS
+            if datapoints.BoundingBox in info.kernels and info.dispatcher is not F.convert_format_bounding_box
```

Comment:
We need to exclude `F.convert_format_bounding_box` since its only job is to change the format.